.ci/bwcVersions

```diff
@@ -1,49 +1,4 @@
 BWC_VERSION:
-  - "7.0.0"
```

Comment:
I don't think all of these should be removed?

Reply:
It was the gradle updateCIBwcVersions command that removed them. I do not know how it makes those decisions, but was curious about this.